mailbox/postgres/src/main/java/org/apache/james/mailbox/postgres/PostgresMailboxSessionMapperFactory.java

```diff
@@ -115,11 +116,12 @@ public PostgresModSeqProvider getModSeqProvider(MailboxSession session) {
 
     @Override
     public AttachmentMapper createAttachmentMapper(MailboxSession session) {
-        throw new NotImplementedException("not implemented");
+        PostgresAttachmentDAO postgresAttachmentDAO = new PostgresAttachmentDAO(executorFactory.create(session.getUser().getDomainPart()), blobIdFactory);
+        return new PostgresAttachmentMapper(postgresAttachmentDAO, blobStore);
     }
 
     @Override
     public AttachmentMapper getAttachmentMapper(MailboxSession session) {
-        throw new NotImplementedException("not implemented");
+        return createAttachmentMapper(session);
```

Comment:
A bit unsure here, 
both create and get methods will create new attachmentMapper.